JavaAssignment/src/prachiJ/AreaPerimeterFind.java

```diff
@@ -0,0 +1,36 @@
+package prachiJ;
+
+import prachiJ.AreaPerimeterFind;
+
+public class AreaPerimeterFind {
+
+	void rectangle(float a,float b)
```

Comment:
Using this method name I am not getting what are you going to perform in this method. So here name should be like areaOfRectangle, Right ?

Do this change for others methods.

Reply:
Thank you Krishna for the feedback. 
I have have added comments, changed method name and did formating.